### PR TITLE
Fix XMIR to EO data conversion

### LINKED ISSUE #2750: XMIRTest.java:49-54: Xmir samples are not converted...
The puzzle `2729-2c784c24` from #2729 has to be resolved: 

https://github.com/objectionary/eo/blob/44724edad3564d1bf9847445cd492ed94939cb6c/eo-parser/src/test/java/org/eolang/parser/XMIRTest.java#L49-L54

The puzzle was created by maxonfjvipon on 28-Dec-23. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by c71n93**
Closes #2750

---

### Detailed summary

- Added new imports in XMIR.java
- Modified the toEO() method in XMIR.java
- Added a new test method in StEoLoggedTest.java
- Modified the xmir-to-eo.xsl file
- Modified the StUnhex.java file
- Added new imports in XMIRTest.java
- Modified the parse() method in XMIRTest.java
- Added new test methods in XMIRTest.java
- Modified the StUnhexTest.java file

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by c71n93** [comment]
@maxonfjvipon could you check this one, please?

File: eo-parser/src/main/resources/org/eolang/parser/xmir-to-eo.xsl [hunk 1]
```diff
@@ -142,17 +142,10 @@ SOFTWARE.
         <xsl:text>"""</xsl:text>
       </xsl:when>
       <xsl:when test="@base='bool'">
```
Comment (maxonfjvipon) [inline eo-parser/src/main/resources/org/eolang/parser/xmir-to-eo.xsl:144]: @c71n93 let's combine `bool`, `int` and `float` under one `when` condition

**Review by maxonfjvipon — CHANGES_REQUESTED** [review]
@c71n93 good one, just one comment from my side

**Comment by c71n93** [comment]
@yegor256 could you check this one,  please?

File: eo-parser/src/test/java/org/eolang/parser/XMIRTest.java [hunk 2]
```diff
@@ -83,12 +84,12 @@ void printsToEO(final String src) throws Exception {
      * @throws IOException If fails
      */
     private static XML parse(final String source) throws IOException {
-        return new XSLDocument(
-            XMIRTest.class.getResourceAsStream("wrap-method-calls.xsl")
-        ).with(new ClasspathSources()).transform(
-            new EoSyntax(
-                "test", new InputOf(source)
-            ).parsed()
+        return new Xsline(new TrClasspath<>(
```
Comment (yegor256) [inline eo-parser/src/test/java/org/eolang/parser/XMIRTest.java:87]: @maxonfjvipon I'm curious, how does this line pass Qulice check? There is obviously an error. Maybe we have Qulice turned off for this module?

Comment (maxonfjvipon) [reply eo-parser/src/test/java/org/eolang/parser/XMIRTest.java:87]: @yegor256 no, we don't, seems like a bug in Qulice. And yes, there's something wrong with indentations. @c71n93 please move `new TrClasspath<>(` to the next line

**Review by maxonfjvipon — APPROVED** [review]

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**